mlos_bench/mlos_bench/services/remote/azure/README.md

```diff
@@ -0,0 +1,37 @@
+# Azure Services
+
+## Authentication
+
+This service allows us to generate tokens for authenticating with other Azure services.
```

Comment:
```suggestion
The `AzureAuthService` allows us to generate tokens for authenticating with other Azure services.

The goal is to be able to support long running experiments ideally without needing prompt the user to refresh their login session to avoid timeouts.
```